dll/cpl/openglcfg/lang/tr-TR.rc

```diff
@@ -1,8 +1,8 @@
-/* TRANSLATOR: 2017, 2020 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
+// COPYRIGHT: 2017-2020 Erdem Ersoy
```

Comment:
```suggestion
// COPYRIGHT: 2017-2020 Erdem Ersoy <erdemersoy@erdemersoy.net>
```

Reply:
No, this email address is outdated. Erdem Ersoy changed his email address about 3 or 4 times over the last 4 years (as often as others do change their underwear). And this one is outdated already and not longer active.